src/typesv2.ts

```diff
@@ -0,0 +1,77 @@
+import type { Id } from './idv2.js';
+
+type OmitStrict<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
+
+export type ValueType = 'TEXT' | 'NUMBER' | 'CHECKBOX' | 'URL' | 'TIME' | 'POINT';
+
+export type ValueOptions = {
+  format?: string;
+  unit?: string;
+};
+
+export type Value = {
+  value: string;
+  options?: ValueOptions;
+};
+
+export type Entity = {
+  id: string;
+  types: string[];
+  properties: Record<string, Value>;
```

Comment:
We might need to align on the shape of `Entity`. In the protobuf I have it defined as an array of `Value`s, where a `Value` looks like this

```ts
type Value = {
  propertyId: string;
  value: string;
  options?: Options;
}
```